packages/v4/patternfly-docs/content/accessibility/application-launcher/application-launcher.md

```diff
@@ -3,16 +3,58 @@ id: Application launcher
+| `hidden` | `.pf-c-app-launcher__menu` | Visually hides the application launcher menu and prevents assistive technologies from navigating/interacting with it. **Required** when `aria-expanded="false"` is passed in. |
+| `aria-disabled="true"` | `.pf-c-app-launcher__menu-item` | Disables the application launcher item, but keeps the item perceivable to users. Use this attribute instead of `disabled` when the item uses an `<a>` element, when you want users to still be aware of the item and that it is disabled, or when you expect or intend for the item to receive focus despite being disabled. For example, if the disabled item has a tooltip, you should pass this prop in to disable it. <br/><br/> When using this attribute, additional steps must be taken via JavaScript to prevent events from firing on the element. |
+| `disabled` | `.pf-c-app-launcher__menu-item` | Disables the application launcher item when the item uses a `<button>` element, preventing it from being perceivable and operable by users navigating via assistive technologies. Use this attribute to disable an item if you don't expect or intend for the item to ever need to receive focus. For example, if the item will never have a tooltip, you should pass this prop in to disable it. |
+| `tab-index="-1"` | `.pf-c-app-launcher__menu-item` | Prevents the application launcher item from receiving focus via keyboard. If the item uses an `<a>` element and you wish to completely disable the item, the `aria-disabled="true"` attribute must also be passed in. |
+| `.pf-screen-reader` | `.pf-c-app-launcher__menu-item span` | Adds text that is accessible only to assistive technologies and is not visually rendered. This element should be added when an application launcher item is expected to open in a new tab/window, and the text content should convey that information. |
+| `alt=""` | `.pf-c-app-launcher__menu-item-external-icon > *`, `.pf-c-app-launcher__menu-item.pf-m-action`, `.pf-c-app-launcher__menu-item-icon` | Removes the icon from the accessibility tree when the icon is an `<img>` element, preventing assistive technologies from potentially announcing duplicate or unnecessary information without visually hiding it. **Required**. |
+| `aria-hidden="true"` | `.pf-c-app-launcher__menu-item-external-icon > *`, `.pf-c-app-launcher__menu-item.pf-m-action`, `.pf-c-app-launcher__menu-item-icon` | Removes the icon from the accessibility tree when the icon is not an `<img>` element, preventing assistive technologies from potentially announcing duplicate or unnecessary information without visually hiding it. **Required**. |
+| `aria-expanded="false"` | `.pf-c-app-launcher__toggle` | Indicates that the application launcher toggle is collapsed to assistive technologies. **Required** if the toggle is collapsed. |
+| `aria-expanded="true"` | `.pf-c-app-launcher__toggle` | Indicates that the application launcher toggle is expanded to assistive technologies. **Required** if the toggle is expanded. |
```

Comment:
Could these two be combined to simplify? Like `aria-expanded=[true|false]` and then say "expanded or collapsed" in the description?

Reply:
Ah definitely! Will update similar instances of that in these files